src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GoToGCPConsoleStep.ts

```diff
@@ -0,0 +1,40 @@
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonOutline } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import Events from 'src/libs/events'
+import * as Utils from 'src/libs/utils'
+import { Step } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/Step'
+import { StepInfo } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepFields'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+interface GoToGCPConsoleStepProps {
+  isActive: boolean
+  stepFinished: () => void
+}
+
+export const GoToGCPConsoleStep = ({ isActive, ...props }: GoToGCPConsoleStepProps) => {
+  return h(Step, { isActive }, [
+    StepHeader({ title: 'STEP 1' }),
+    div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between' } }, [
+      h(StepInfo, { style: { maxWidth: '60%' } },
+        ['Go to the Google Cloud Platform Billing Console and sign-in with the same user you use to login to Terra.']),
+      h(ButtonOutline, {
+        disabled: false,
+        href: 'https://console.cloud.google.com',
```

Comment:
I think https://console.cloud.google.com/billing would be a more useful link here. Changing the link may be out of scope, so it's up to you

Reply:
Happy to change the link-- @admull do you agree with @marctalbott's proposal?

Reply:
Decided with Adam to keep as-is.